### DISCUSSION TITLE
Support SHACL schema as alternative to json-schema

### DISCUSSION THREAD

**Discussion by Fak3** [OAI/sig-moonwalk#120] [Ideas]
It would be nice to allow a resource to be linked with SHACL schema. So that API consumers would know a schema to validate data being sent\received. So that tools like swagger-ui would be able to render appropriate human-friendly data descriptions and playground forms validation.

**Comment by Fak3** [comment]
refs OAI/OpenAPI-Specification#1100

**Comment by MikeRalphson** [comment]
Presumably https://www.w3.org/TR/shacl/?  See also the `x-oas-draft-alternativeSchema` work in OAI/OpenAPI-Specification#1532 which allows this to be supported today.

**Comment by Fak3** [comment]
What is the process to add a new alternative schema to this registry: https://spec.openapis.org/registry/alternative-schema/ ?

**Comment by MikeRalphson** [comment]
As you can probably see, the registry you linked to is still in draft state so the data points are all just examples for now.

That said, the proposed process for adding a registry value should be along the lines of the example PR OAI/OpenAPI-Specification#1811 though the github PR template is not yet set up.
